### PR TITLE
PLP-15: Database seed

### PR DESCRIPTION
**Pull request by Nishan212**

### CODE REVIEW

File: server/prisma/seedData.ts [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+];
+
+export const eligibilities: (Omit<Eligibility, "id" | "branches"> & {
+	branchesAbrr: string[];
+})[] = [
+	{
+		cgpa: 8.5,
+		tenthScore: 8.5,
+		twelfthScore: 8.5,
+		diplomaScore: 8.5,
```
Comment (karan316) [inline server/prisma/seedData.ts:455]: scores should be in terms of %. i.e. 0 to 100. Without the % symbol. eg 88.3, 87, 94.5 etc.

File: server/prisma/seedData.ts [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+		companyAbbr: "FB",
+	},
+];
+
+export const shortlists: (Omit<Shortlist, "id" | "students"> & { studentsRegNo: string[] })[] = [
+	{
+		step: "ELIGIBILITY",
+		studentsRegNo: ["180911111", "180433333"],
+	},
+	{
```
Comment (karan316) [inline server/prisma/seedData.ts:556]: shortlist maps to student with email ID, not registration number.

File: server/src/graphql/typeDefs.ts [hunk 3]
```diff
@@ -127,6 +127,7 @@ const typeDefs = gql`
 	type Company {
 		id: String!
 		name: String!
+		abbreviation: String!
```
Comment (karan316) [inline server/src/graphql/typeDefs.ts:130]: do we need an abbreviation for the company? where is it used?

**Review by karan316 — CHANGES_REQUESTED** [review]

File: server/prisma/seedData.ts (hunk 1 above)
Comment (Nishan212) [reply server/prisma/seedData.ts:455]: Noted

File: server/prisma/seedData.ts (hunk 2 above)
Comment (Nishan212) [reply server/prisma/seedData.ts:556]: emailID is in User model. I'll try a work around this.

File: server/src/graphql/typeDefs.ts (hunk 3 above)
Comment (Nishan212) [reply server/src/graphql/typeDefs.ts:130]: It will be a unique ID for a company, it will be helpful I guess...

Comment (karan316) [reply server/src/graphql/typeDefs.ts:130]: But I don't think we will use it anywhere. We already have the company id for a unique field.

Comment (Nishan212) [reply server/src/graphql/typeDefs.ts:130]: Removed

File: server/prisma/seedData.ts (hunk 2 above)
Comment (Nishan212) [reply server/prisma/seedData.ts:556]: Done.

**Review by karan316 — APPROVED** [review]

**Pull request merged**